microsoft/PowerToys
# [Settings] Make images clickable and fix issue with Power Preview settings image

## Summary of the Pull Request

_What is this about?_

Makes images clickable and fix issue with Power Preview settings image

## PR Checklist
* [x] Applies to #6703 and #8561
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #8526

## Info on Pull Request

_What does this include?_

Make settings images clickable. And also fixes the Power Preview settings image bug (Not showing up when there is no network access)

## Validation Steps Performed

_How does someone test & validate?_

- Build and click on "Learn more" and the image itself.
- Turn off network access and see if the Power Preview image loads
